artifactory/services/security_test.go

```diff
@@ -0,0 +1,30 @@
+package services
+
+import "testing"
+
+func TestBuildCreateTokenUrlValuesExpiresIn(t *testing.T) {
```

Comment:
I suggest merging this test with the other tokens tests in *artifactorysecurity_test.go*.
Let's also add a description for this test, explaining what it being tested.

Reply:
Currently the tests are testing the unexported method `buildCreateTokenUrlValues` in [security.go](url). I think migrating them to [artifactorysecurity_test.go](https://github.com/jfrog/jfrog-client-go/blob/dev/tests/artifactorysecurity_test.go) would mean to build integration tests, which need a local artifactory instance.